Gems/PhysX/Code/Tests/CharacterControllerTests.cpp

```diff
@@ -159,6 +159,115 @@ namespace PhysX
         }
     }
 
+    TEST_F(PhysXDefaultWorldTest, CharacterController_GamePlayComponenet_EntityFallsUnderGravity)
+    {
+        PhysX::CharacterControllerComponent* characterComponent = nullptr;
+        PhysX::CharacterGameplayComponent* gameplayComponent = nullptr;
+        AzFramework::TransformComponent* transform = nullptr;
+
+        float expectedGravity = 1.5f;
```

Comment:
nitpick - I would call this expectedGravityMultiplier, otherwise it might read like this value is the actual acceleration.

Reply:
Updated